src/main/java/org/gridsuite/modification/server/modifications/ModificationUtils.java

```diff
@@ -527,6 +528,27 @@ public Terminal getTerminalFromIdentifiable(Network network, String equipmentId,
         return null;
     }
 
+    public List<Terminal> getTerminalsFromIdentifiable(Identifiable<?> identifiable) {
+        if (identifiable instanceof Branch<?> branch) {
+            return Stream.of(
+                    branch.getTerminal1(),
+                    branch.getTerminal2()
+            ).collect(Collectors.toList());
+        } else if (identifiable instanceof ThreeWindingsTransformer w3t) {
+            return Stream.of(
+                    w3t.getLeg1().getTerminal(),
+                    w3t.getLeg2().getTerminal(),
+                    w3t.getLeg3().getTerminal()
+            ).collect(Collectors.toList());
+        } else if (identifiable instanceof HvdcLine hvdcLine) {
```

Comment:
How about TieLine, DanglingLine?

Reply:
Out of the scope for the moment
Only branch, 3wt and HVDC